templates/keras_cv/index.md

```diff
@@ -0,0 +1,48 @@
+# KerasCV
+
+---
+## Quick links
+
+* [Getting started with KerasTuner](/guides/keras_tuner/getting_started/)
+* [KerasTuner developer guides](/guides/keras_tuner/)
+* [KerasTuner API reference](/api/keras_tuner/)
+
+
+---
+## Installation
+
+KerasCV requires **Python 3.6+** and **TensorFlow 2.9+**.
+
+Install the latest release:
+
+```
+pip install keras-cv --upgrade
+```
+
+You can also check out other versions in our
+[GitHub repository](https://github.com/keras-team/keras-tuner).
```

Comment:
Update to keras-cv, maybe link directly to `/releases` as well?

Reply:
Done, good call.